cfme/tests/intelligence/reports/test_canned_corresponds.py

```diff
@@ -27,6 +28,26 @@ def compare(db_item, report_item):
         return db_item is None and report_item == ''
 
 
+@pytest.fixture(scope="module")
```

Comment:
Required: Please update paginator to pagination = View.nested(NonJSPaginationPane) for Details view, as JSPaginationPane is not working there